<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QDialog" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main menu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="jugar">
       <property name="text">
        <string>Jugar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrar_hist">
       <property name="text">
        <string>Borrar historial</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
